vault/identity_store.go

```diff
@@ -247,6 +266,8 @@ func (i *IdentityStore) Invalidate(ctx context.Context, key string) {
 
 		txn.Commit()
 		return
+	case strings.HasPrefix(key, oidcTokensPrefix):
```

Comment:
Super minor nitpick, can you put a space before this case statement? For non-trivial cases I find it helps readability, and the other cases in this switch already have it.

Reply:
Done